internal/apps/unit-of-work/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+  app:
+    build:
+      context: ../../.. # repository root
```

Comment:
NB: I wondered why this was needed, as opposed to just doing `build: .`. I realized that it's so the local copy of dd-trace-go is available, since the app points to it with a `replace` directive.

Reply:
Will add a comment